kong/templates/kong_defaults.lua

```diff
@@ -39,6 +39,7 @@ client_body_buffer_size = 8k
 error_default_type = text/plain
 
 nginx_http_ssl_protocols = TLSv1.1 TLSv1.2 TLSv1.3
+nginx_http_ssl_prefer_server_ciphers = on
```

Comment:
Should the default be `off`, given the default for `ssl_cipher_suite` is `intermediate` and in that case the client chooses?

Reply:
See: https://github.com/mozilla/server-side-tls/issues/260

I think we should include `prefer_server_ciphers` as part of the configuration bundle too, and only enable it for `old` configurations.

Reply:
In fact I think that's what @bungle did already in https://github.com/Kong/kong/pull/5342/files#diff-a2237d21dd96315727f1b771bf0111f5R479, ignore last comment.

Reply:
@gszr rationale for this being default is that when user wants to use `custom` cipher suite, it will default to `prefer cipher suites`. We can perhaps change that later. But for older suites it was generally preferred. This default is ignored with `modern`, `intermediate` and `old`, see: https://github.com/Kong/kong/pull/5342/files#diff-a2237d21dd96315727f1b771bf0111f5R22-R74

Reply:
... and if you are not using `older` suite, you probable should then consider using `intermediate` or `modern` directly without trying to specify your own.

Reply:
I am ok to change the default to `off` if we think that it is more common use case. Let me know. Above was my reasoning for that (also it was the old default).

Reply:
I think we should leave it to `on` by default for `custom`, as in most cases that may result in a more secure configuration mode.